polaris/ocean/tests/global_convergence/cosine_bell/__init__.py

```diff
@@ -129,12 +129,13 @@ def _setup_steps(self, config):
             subdir = f'{mesh_name}/map'
             viz_map = VizMap(test_case=self, name=name, subdir=subdir,
                              mesh_name=mesh_name)
-            self.add_step(viz_map)
+            self.add_step(viz_map, run_by_default=False)
```

Comment:
@sbrus89, since you and @cbegeman were discussing this today, I'm turning off the viz steps in `cosine_bell` with this parameter.  Could you just check the code by eye and see if it looks okay.  I think my testing is sufficient.